### ISSUE TITLE
[Bug]: Slide hanging if overdragged/scrolled 

### ISSUE THREAD

**Issue by DanielSpindler** [davidjerleke/embla-carousel#785]
### Which variants of Embla Carousel are you using?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Steps to reproduce

The bug occurs when I "overscroll/drag" the slides

i got an example sandbox https://codesandbox.io/p/sandbox/embla-carousel-default-react-ww73yq?file=%2Fsrc%2Fjs%2FEmblaCarouselArrowButtons.tsx from the docs and tried it there to reproduce.

https://github.com/davidjerleke/embla-carousel/assets/71325321/e76122c3-c6d6-4d2b-8763-1f00f24ad095

### Expected Behavior

Jumping back

### Additional Context

I have also tried :
Google Chrome
Firefox 
Safari

And also tried to implement a "overscroll" workaround from the issues which sadly jumped around, but not related.

I've searchd the issues and docs throughly and found nothing that could be directly related.

### Version

v8.0.0

### CodeSandbox

https://codesandbox.io/p/sandbox/embla-carousel-default-react-ww73yq?file=%2Fsrc%2Fjs%2FEmblaCarouselArrowButtons.tsx

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for bug reports

**Comment by DanielSpindler** [comment]
Thanks alot and no Problem! 

Greetings,
Daniel

**Comment by chiumax** [comment]
hey guys, running into the same issue with the `WheelGesturesPlugin` here's a snippet that sort of fixes this issue:
```js
 embalaApi.on('settle', (api: CarouselApi) => {
        if (!api) {
          return;
        }
        const { limit, location, scrollTo } = api.internalEngine();
        if (location.get() > limit.max) {
          scrollTo.index(0, 0);
        }
        if (location.get() < limit.min) {
          scrollTo.index(embalaApi.scrollSnapList().length - 1, 1);
        }
      });
```

Ideally it would be nice to reset the momentum early so it never gets to this or take so long to rubber band back.

**Comment by tayteboss** [comment]
Hey @davidjerleke.

On this thread of overdrag/scroll. Is there a way to stop users from being able to overscroll/drag entirely? So if you hit the start/end of the carousel it's a hard wall and that slide becomes stuck?

Thanks

**Comment by davidjerleke** [comment]
@tayteboss your request isn’t relevant in this issue and already exists here:

- #42

**Comment by davidjerleke** [comment]
@DanielSpindler and/or @sarussss, 

Can you reproduce this problem in [this CodeSandbox](https://qvkkhs.csb.app/)?

**Comment by sarussss** [comment]
Hi @davidjerleke 
I checked, the problem I had before has been **resolved**, **great**!

**Comment by DanielSpindler** [comment]
https://codesandbox.io/p/sandbox/embla-carousel-default-react-forked-vcjj84?file=%2Fsrc%2Fjs%2FEmblaCarouselArrowButtons.tsx updated the other CodeSandBox for now @davidjerleke, @sarussss seems like it is not gone,  will update it in a live app today and give a feedback after

**Comment by davidjerleke** [comment]
@DanielSpindler did you try the CodeSandbox in [this comment](https://github.com/davidjerleke/embla-carousel/issues/785#issuecomment-2119900731)? Because a **fix is not released yet**!

**Comment by DanielSpindler** [comment]
@davidjerleke my fault ive been reading it wrong sorry! do you have a version with 
 "embla-carousel-wheel-gestures": "^8.0.0-rc05" 
 cant reproduce with dragging anymore

**Comment by davidjerleke** [comment]
@DanielSpindler you can fork the Sandbox and add the Wheel Gestures plugin and test it if you want.

**Comment by DanielSpindler** [comment]
@davidjerleke https://codesandbox.io/p/sandbox/bug-785-forked-psk7k8?file=%2Fsrc%2Fjs%2Fembla-carousel.js%3A1857%2C25-1857%2C30 seems to work just fine, cant reproduce anymore

**Comment by davidjerleke** [comment]
Thanks @sarussss and @DanielSpindler. I'll throw a comment in here once the bug fix has been released.

**Comment by davidjerleke** [comment]
A bug fix for this will be included in the next release.

**Comment by davidjerleke** [comment]
@sarussss and @DanielSpindler a bug fix for this was just released in [v8.1.4](https://github.com/davidjerleke/embla-carousel/releases/tag/v8.1.4).
